vita3k/gui/src/gui.cpp

```diff
@@ -189,10 +189,8 @@ void load_game_background(GuiState &gui, HostState &host, const std::string &tit
             vfs::read_app_file(buffer, host.pref_path, title_id, "sce_sys/livearea/contents/bg.png");
             vfs::read_app_file(buffer, host.pref_path, title_id, "sce_sys/livearea/contents/bg0.png");
         } else {
-            if (gui.user_backgrounds.find(host.cfg.background_image) != gui.user_backgrounds.end())
+            if (gui.current_background != gui.user_backgrounds[host.cfg.background_image])
```

Comment:
Was the `.find()` bad? It seems kinda important...

Reply:
look after for understund :), don't need find anymore :)